lib/universal_peripherial.dart

```diff
@@ -77,89 +73,113 @@ class _State {
     _context = context;
   }
 
+  Central get central => _context.central;
+
+  bool isFeatureSupported(String feature) {
+    return _context.features.contains(feature);
+  }
+
+  void send(String command) {
+    this._context.send(command);
+  }
+
+  void transitionTo(_State nextState) {
+    this._context.transitionTo(nextState);
+  }
+
   void onEnter() {}
 
   void onPeripheralCmd(String cmd) {
-    if (this._context.features.msg && cmd.startsWith("msg")) {
-      this._context.central.showMsg(getCommandParams(cmd));
-      this._context.send('ok');
+    if (cmd.startsWith("msg") && isFeatureSupported("msg")) {
+      this.central.onPeripheralMsg(getCommandParams(cmd));
```

Comment:
is `this.` needed ? Same everywhere

Reply:
just preference. Shall I remove it?

Reply:
Sure. It doesn't provide any usable information, but add a lot of repeating text in code